### DISCUSSION TITLE
[Code Search and Code View Beta] Code search doesn't work

### DISCUSSION THREAD

**Discussion by valentineus** [community/community#40337] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Greetings!

Recently gained access to the new GitHub Search (Code Search and Code View Beta). The code search doesn't work. Shows an error: **"Search failed. Please try again later."**. I tried different values in the search input.

HTTP request returns data:

```json
// GET https://github.com/search?q=Hello%2C+World%21
{
  "locale": "en",
  "payload": {
    "elapsed_millis": 22,
    "errors":
    [
      {
        "message": "Search failed. Please try again later."
      }
    ],
    "facets":
    [],
    "header_redesign_enabled": false,
    "page": 0,
    "page_count": null,
    "protected_org_logins":
    [],
    "result_count": 0,
    "results":
    [],
    "topics": null,
    "type": "code"
  },
  "title": "Code search results"
}
```

![github com_search_q=Hello%2C+World%21 type=code(Desktop)](https://user-images.githubusercontent.com/2735807/204496604-4988bbf9-c7a9-471c-8d45-76026072a232.png)

P.S. I hope I didn't make a mistake about where the error was posted😅.

**Comment by valentineus** [comment]
I'm using browser Ungoogled Chromium (107.0.5304.121, Official Build). Of course to check, I disabled the ad blocking and cleaned the browser cache.

**Comment by ossanna16** [comment]
Hi there @valentineus and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by look** [comment, marked as answer]
@valentineus this should be resolved. We had a brief period of downtime yesterday.
